opba-protocols/xs2a-protocol/pom.xml

```diff
@@ -193,6 +193,12 @@
             <artifactId>spring-web-test-client</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Protocol should not depend directly on DB module

Reply:
Done